justfile

```diff
@@ -0,0 +1,3 @@
+run-playground:
+  cargo build --features flashblocks --bin op-rbuilder -p op-rbuilder
```

Comment:
Why only flashblocks?

Reply:
removed flashblocks from this iteration.